### ISSUE TITLE
Réglement intérieur : Membres honorifiques

### ISSUE THREAD

**Issue by nfroidure** [francejs/association#3]
Je pense que le choix des membres honorifiques doit être fait lors des AG par les membres et non par le bureau. En revanche, pour moi, c'est à vie ou rien.

Je mettrai : 
"Les membres honorifiques sont proposés par le bureau ou un groupe d'au moins vingt adhérents par courrier, signés de chacun d'entre eux et remis au bureau. La validation des soumissions a lieu lors de l'assemblée générale et doit obtenir l'approbation de plus des deux tiers des votants.

Le statut de membre honorifique ne peut être révoqué. Les membres honorifiques sont exempts de cotisation, mais peuvent s'en acquitter sur la base du volontariat. Les membres honorifiques disposent du droit de vote uniquement s'ils s'acquittent de leur cotisation."

Pour moi, le statut de membre honorifique ne doit pas être accessible à tous, je vois bien des personnes comme Brendan Eich, John Resig etc... faut vraiment que la personne fasse l'unanimité et ai vraiment pesé sur le développement de JavaScript. Ça me paraît aussi important que ce soit la communauté qui décide de ça plutôt que le bureau qui pourrait être tenté de faire du copinage. Pour le droit de vote, je pense aussi que c'est important que les membres honorifiques n'aient pas le droit de votre s'il ne s'acquittent pas de leur cotisation.

**Comment by MathRobin** [comment]
Tu veux parler de "Membres honoraires" plutôt non ?

**Comment by t8g** [comment]
Mouais ... personnellement j’enlèverai cet article.
Je ne vois pas bien l'intérêt ni pour le membre ni pour l'asso.

**Comment by nfroidure** [comment]
Les statuts AFUP parlent de membres honorifiques selon la définition wikipédia "membre honoraire" veut dire la même chose.

**Comment by nfroidure** [comment]
t8g : à vrai dire, effectivement, tout le monde connaît les personnes que je cite, donc effectivement, ça me semble plus un appareil de copinage qu'un truc vraiment utile pour l'asso. Supprimer est un option aussi.

**Comment by MathRobin** [comment]
@nfroidure au temps pour moi.

En soit, je ne vois pas trop l’intérêt si c'est pour élire des personnes déjà plus que connues. Mais quitte à être FranceJS, autant faire une élection à l'année d'une personnalité française du JS. Ça serait plus cohérent et utile. Je pense.
Suggestion effectuée.

**Comment by nfroidure** [comment]
Je suis d'accord avec la notion de mise à l'honneur, faut juste trouver une forme qui ne favorise pas la cooptation.

On pourrait élire le projet de l'année ou le développeur de l'année...

**Comment by AMorgaut** [comment]
A partir du moment ou on créer FranceJS je pense que l'un des objectif, peut être non dit pour l'instant, sera de promouvoir l'expertise française dans cette technologie, auquel cas, si ce n'est sous la forme de membre honoraires ou honorifiques, il peut être bon  de formuler un moyen de mettre en avant quelques spécialistes.

Le fait que cela passe par l'AG plutôt que par le bureau est une première protection contre le copinage. A voir si l'on peut organiser une election annuelle plus global impliquant les groupes d'utilisateurs français même si je crains un peu une revendication du poids des groupes (plus de voix pour ParisJS?)
L'autre option étant une election ouverte à tous lors d'une conference officielle FranceJS ou sur le Web (exposés au hacking).

Au dela de l'option AG, j'ai du mal a croire à une option élection virtuelle sur le Web. côté conférence j'imagine aujourd'hui un soutien potentiel de dotJS ou JS.everywhere() Europe... A voir...
